contrib/pmsnare/pmsnare.c

```diff
@@ -70,8 +72,9 @@ DEFobjCurrIf(datetime)
 
 
 /* static data */
-static int bParseHOSTNAMEandTAG;	/* cache for the equally-named global param - performance enhancement */
-
+static int bParseHOSTNAMEandTAG;  /* cache for the equally-named global param - performance enhancement */
+static char tabRepresentation[5];
```

Comment:
It's set in endCnfLoad. Once the rsyslog config has been fully loaded, it will use the global settings and override them as needed with instance-specific settings, then set this.